packages/engine/Source/Scene/DynamicEnvironmentMapManager.js

```diff
@@ -717,17 +737,19 @@ function updateIrradianceResources(manager, frameState) {
   const dimensions = irradianceTextureDimensions;
 
   let texture = manager._irradianceMapTexture;
-  if (!defined(texture)) {
-    texture = new Texture({
-      context: context,
-      width: dimensions.x,
-      height: dimensions.y,
-      pixelDatatype: PixelDatatype.FLOAT,
-      pixelFormat: PixelFormat.RGBA,
-    });
-    manager._irradianceMapTexture = texture;
+  if (defined(texture)) {
```

Comment:
Should this be checking `isDestroyed()` first? Some places in this PR you do and some you don't. One of these might be the cause of the error I mentioned above.

Also worth noting that this is exactly why I had questions about this pattern https://github.com/CesiumGS/cesium/issues/12266

Reply:
Throughout this class, I've looked to set any destroyed property back to `undefined` to avoid the need for the check. 

I think https://github.com/CesiumGS/cesium/issues/12266 is still worth considering, but I think for the purposed of this PR and to keep the scope small I'd like to avoid addressing it here.